src/app/core.ts

```diff
@@ -83,6 +84,16 @@ export interface AppOptions {
    * specifying an HTTP Agent in the corresponding factory methods.
    */
   httpAgent?: Agent;
+
+  /**
```

Comment:
Probably add more details on the meaning of the fields in RetryConfig? E.g. how to interpret backoffFactor